chroma_core/models/host.py

```diff
@@ -1685,7 +1684,7 @@ def get_steps(self):
                         ReplaceNidsStep,
                         {
                             "target": "%s" % target,
-                            "nids": target.filesystem.mgs.nids(),
+                            "nids": target.best_available_host().lnet_configuration.get_nids(),
```

Comment:
Won't this only provide the nids for the primary or mounted host?  What about the secondary host?

Reply:
updated